packages/vite-plugin-cloudflare/package.json

```
@@ -0,0 +1,65 @@
+{
+	"name": "@cloudflare/vite-plugin",
+	"version": "0.0.0",
+	"description": "Cloudflare plugin for Vite",
+	"keywords": [
+		"cloudflare",
+		"workers",
+		"cloudflare-workers",
+		"vite",
+		"vite-plugin"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/flarelabs-net/vite-plugin-cloudflare",
+		"directory": "packages/vite-plugin-cloudflare"
+	},
+	"license": "MIT",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": "./dist/index.js"
+		}
+	},
+	"main": "./dist/index.js",
+	"types": "./dist/index.d.ts",
+	"files": [
+		"dist"
+	],
+	"scripts": {
+		"build": "tsup",
+		"check:type": "tsc --build",
+		"test:ci": "vitest run",
+		"watch": "tsup --watch"
+	},
+	"dependencies": {
+		"@hattip/adapter-node": "^0.0.49",
+		"miniflare": "workspace:*",
+		"unenv": "catalog:vite-plugin",
+		"ws": "^8.18.0"
+	},
+	"devDependencies": {
+		"@cloudflare/workers-shared": "workspace:*",
+		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@cloudflare/workers-types": "^4.20241230.0",
+		"@types/node": "catalog:vite-plugin",
+		"@types/ws": "^8.5.13",
+		"magic-string": "^0.30.12",
+		"tsup": "^8.3.0",
+		"typescript": "catalog:vite-plugin",
+		"vite": "^6.0.3",
+		"wrangler": "workspace:*"
+	},
+	"peerDependencies": {
+		"vite": "^6.0.3",
+		"wrangler": "^3.94.0"
```

This version of `wrangler` is too old. The minimum we support is `3.101.0`. Ideally it should match the workspace version.

```suggestion
		"wrangler": "^3.101.0"
```

---

Changing it to workspace:^ so that users can use newer versions of Wrangler without updating this version. OK?